projects/models.py

```diff
@@ -52,6 +52,18 @@ def is_overdue(self):
         today = date.today()
         return today > self.due_date and self.status != Project.ProjectStatus.COMPLETED
 
+    def set_to_done(self):
+        "Sets all the features associated with it to done, returns false if no features"
+        Project.objects.filter(id=self.id).update(status=30)
```

Comment:
Instead of querying the database for all Projects and filtering by id, this is already inside the Project model -- the Project is `self`

Reply:
Double check the syntax but it would be something like this
```suggestion
        self.update(status=Project.ProjectStatus.COMPLETED)
```

Reply:
Not allowed to update and instance

Reply:
I'm going to leave this line in for now, and make the other changes you recommended.

Reply:
nevermind the self.save() command works for instances (self)

Reply:
Here's a relevant section of the documentation:
https://docs.djangoproject.com/en/3.0/ref/models/instances/#updating-attributes-based-on-existing-fields

Something like this (syntax may be slightly different)
`self.status = Project.ProjectStatus.COMPLETED
self.save()`